specs/core/0_beacon-chain.md

```diff
@@ -855,76 +850,99 @@ def split(values: List[Any], split_count: int) -> List[Any]:
     ]
 ```
 
+#### `get_committees_per_slot`
+
+```python
+def get_committees_per_slot(active_validator_count: int) -> int:
+    return max(
+        1,
+        min(
+            SHARD_COUNT // EPOCH_LENGTH,
+            len(active_validator_indices) // EPOCH_LENGTH // TARGET_COMMITTEE_SIZE,
```

Comment:
typo, it should just be `active_validator_count // EPOCH_LENGTH // TARGET_COMMITTEE_SIZE`